lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -564,3 +568,41 @@ func getRestrictedImeis(imeis []*lte_models.Imei) []*lte_mconfig.MME_ImeiConfig
 	}
 	return ret
 }
+
+func getNProbeConfig(networkID string) (*lte_mconfig.LIAgentD, *lte_mconfig.PipelineD_LiUes) {
+	liUes := &lte_mconfig.PipelineD_LiUes{}
+	liAgent := &lte_mconfig.LIAgentD{
+		LogLevel: protos.LogLevel_INFO,
+	}
+	ents, _, err := configurator.LoadAllEntitiesOfType(
+		networkID,
+		lte.NetworkProbeTaskEntityType,
+		configurator.EntityLoadCriteria{LoadConfig: true},
+		serdes.Entity,
+	)
+	if err != nil {
+		return liAgent, liUes
+	}
+
+	for _, ent := range ents {
```

Comment:
maybe this whole function should in to the nprobe_models. Similar to what we do at `feg_models.ToFederatedModesMap()`